pynucastro/networks/rate_collection.py

```diff
@@ -506,54 +506,63 @@ def get_screening_map(self):
             return screening_map
 
         for r in self.rates:
-            screen_nuclei = r.ion_screen
-            if self.symmetric_screening:
-                screen_nuclei = r.symmetric_screen
+            all_screen_nuclei = []
+            if isinstance(r, ApproximateRate):
+                # loop over all of the rates in the approximate rate and grab their
+                # screening nuclei and append to the list
+                raise NotImplementedError("haven't writtne this yet")
+            else:
+                screen_nuclei = r.ion_screen
+                if self.symmetric_screening:
+                    screen_nuclei = r.symmetric_screen
+                all_screen_nuclei.append(screen_nuclei)
 
-            if screen_nuclei:
-                nucs = "_".join([str(q) for q in screen_nuclei])
+            for screen_nuclei in all_screen_nuclei:
+                # screen_nuclei may be [] if it is a decay, gamma-capture, or neutron-capture
+                if screen_nuclei:
```

Comment:
This would be simpler as
```suggestion
                if not screen_nuclei:
                    continue
```

Reply:
agreed